packages/api/amplify_api_dart/lib/src/graphql/web_socket/blocs/web_socket_bloc.dart

```diff
@@ -264,9 +264,7 @@ class WebSocketBloc with AWSDebuggable, AmplifyLoggerMixin {
   // Init connection and add channel events to the event stream.
   Stream<WebSocketState> _init(InitEvent event) async* {
     assert(
-      _currentState is DisconnectedState ||
-          _currentState is ConnectingState ||
-          _currentState is ReconnectingState,
+      _currentState is InitializingState || _currentState is ReconnectingState,
```

Comment:
question: Am I correct in understanding we no longer expect to get here while in `ConnectingState`? Seems like this is a branch we removed from here but seems like login in _registration prevents it.

Reply:
Discussed offline.